pkg/apis/cnsoperator/config/cns.vmware.com_storagepolicyusages.yaml

```diff
@@ -107,5 +107,3 @@ spec:
         type: object
     served: true
     storage: true
-    subresources:
```

Comment:
With status added as subResource, it is preventing operations like modify/update/add on status field.
Hence this PR is removing Subresources for testing

Reply:
Something similar to https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/553

Reply:
Have you tried "patch"?  We should be using "patch" instead of "update" as "update" causes lots of API update conflicts.

Reply:
I have changed the logic to perform patch operation. But, we still need to remove this subresources field for my testing